src/proxy.js

```diff
@@ -107,6 +107,14 @@ const objectTraps = {
 			// Store drafts on the copy (when one exists).
 			drafts = state.copy
 		}
+		
+		if (!Object.keys(state.base).includes(prop)) {
+			return value
+		}
+
+		if (!Object.keys(state.base).includes(prop)) {
+			return value
+		}
```

Comment:
Are my eyes deceiving me, or are there two exact copies of the keys-check + return code? (typo?)

Reply:
Thanks